app/src/main/java/co/icanteach/projectx/InterviewApplication.kt

```diff
@@ -19,11 +24,29 @@ class InterviewApplication : Application(), HasActivityInjector {
 
     override fun onCreate() {
         super.onCreate()
+        Stetho.initialize(
```

Comment:
We should provide more abstract tools layer. StethoTool, XLibraryTool, YLibraryTool. We can discuss on this topic if you want to implement this kind of feature.

After the implementation, all we should do is to use Tools.init()